src/components/builder.ts

```diff
@@ -270,7 +270,8 @@ export class Builder implements IBuilder {
                 command += ' ' + args[0]
             }
             this.extension.logger.addLogMessage(`cwd: ${path.dirname(rootFile)}`)
-            this.currentProcess = cs.spawn(command, [], {cwd: path.dirname(rootFile), env: envVars, shell: getShell()})
+            // The shell option below should be true. Cannot recall why. Need to check. @James @22-Sep-14
```

Comment:
This is because, magic comments define the command + its arguments in a single variable

https://github.com/James-Yu/LaTeX-Workshop/wiki/Compile#magic-comments